### DISCUSSION TITLE
Round Stack corner

### DISCUSSION THREAD

**Discussion by miranyx** [chakra-ui/chakra-ui#5452] [Q&A]
Hello,
How can I round the Stack corners I tried 
```js 
<Stack
                direction={{ base: 'column', md: 'row' }}
                display={{ base: 'none', md: 'flex' }}
                width={{ base: 'full', md: 'auto' }}
                alignItems="center"
                mt={{ base: 4, md: 0 }}
                flexGrow={1}
                border-radius="10%"
 >
```
but the corner just won't get round.

**Comment by dmitriifrlv** [comment]
You can add border-radius property with [sx prop](https://chakra-ui.com/docs/features/the-sx-prop) :
```
    <Stack
      sx={{ borderRadius: "10%" }}
    >
```

**Reply by nikolovlazar** [reply]
A better way would be to use the `borderRadius` directly as prop, instead of using the `sx` prop. Or, you can use the `rounded` prop too.

**Comment by nikolovlazar** [comment]
@benni347 you should use camel case when adding style props, so instead of `border-radius` you should do `borderRadius`. The `rounded` prop does that too.
